src/index.jsx

```diff
@@ -1,14 +1,32 @@
 import React from 'react';
 import { render } from 'react-dom';
+import { Box } from '@material-ui/core';
 import { AppContainer } from 'react-hot-loader';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import CssBaseline from '@material-ui/core/CssBaseline/CssBaseline';
+import NavBar from './components/Navbar';
 import App from './App';
+import About from './components/About';
 
 const root = document.getElementById('root');
 const load = () => render(
   (
-    <AppContainer>
-      <App />
-    </AppContainer>
+    <Router>
```

Comment:
To use hashes, I think you'll want to use HashRouter here, instead of just Router (I think.. I'm not sure)

Reply:
[According to this](https://github.com/ReactTraining/react-router/issues/394#issuecomment-220221604), we need to use BrowserRouter in order to work with this package. It is built upon BrowserRouter to work with all sorts of routes. 

> When you click on a link created with react-router-hash-link it will scroll to the element on the page with with the id that matches the #hash-fragment in the link. Note that you must use React Router's BrowserRouter for this to work.

> It uses a Router onUpdate hook and calls element.scrollIntoView() if a #hash is present in the url. Since it's defined on the Router component, the scroll functionality will work with every route, and it works when linking to a #hash on the current route, or when linking to a #hash while navigating to a different route.